js/packages/openinference-instrumentation-langchain/package.json

```diff
@@ -7,21 +7,43 @@
   "module": "dist/esm/index.js",
   "esnext": "dist/esnext/index.js",
   "types": "dist/src/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Arize-ai/openinference.git"
+  },
   "scripts": {
     "prebuild": "rimraf dist & pnpm run version:update",
     "build": "tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
     "version:update": "../../scripts/version-update.js",
-    "type:check": "tsc --noEmit"
+    "type:check": "tsc --noEmit",
+    "test": "jest --setupFiles dotenv/config"
   },
   "keywords": [],
+  "files": [
+    "dist",
+    "src"
+  ],
   "author": "oss-devs@arize.com",
   "license": "Apache-2.0",
   "dependencies": {
     "@arizeai/openinference-semantic-conventions": "workspace:*",
     "@opentelemetry/api": "^1.7.0",
-    "@opentelemetry/instrumentation": "^0.46.0"
+    "@opentelemetry/core": "^1.23.0",
+    "@opentelemetry/exporter-trace-otlp-proto": "^0.50.0",
+    "@opentelemetry/instrumentation": "^0.46.0",
+    "@opentelemetry/resources": "^1.19.0",
+    "@opentelemetry/semantic-conventions": "^1.19.0"
```

Comment:
these need to be moved to devDependancies. If these are dependancies, it will cause separate copies to be installed for this module.

Reply:
Oh gotcha, i see some of these as direct deps for openai instrumentation, how do we determine when they should be direct vs. dev deps?